scripts/start.sh

```diff
@@ -4,14 +4,35 @@ set -e
 # Dafault inputs.
 KEEP_ETHEREUM_PASSWORD_DEFAULT="password"
 LOG_LEVEL_DEFAULT="info"
-CONFIG_FILE_PATH_DEFAULT=$(realpath -m $(dirname $0)/../config.toml)
+CONFIG_DIR_PATH_DEFAULT=$(realpath -m $(dirname $0)/../)
 
 # Read user inputs.
 read -p "Enter ethereum accounts password [$KEEP_ETHEREUM_PASSWORD_DEFAULT]: " ethereum_password
 KEEP_ETHEREUM_PASSWORD=${ethereum_password:-$KEEP_ETHEREUM_PASSWORD_DEFAULT}
 
-read -p "Enter path to keep-core client config [$CONFIG_FILE_PATH_DEFAULT]: " config_file_path
-CONFIG_FILE_PATH=${config_file_path:-$CONFIG_FILE_PATH_DEFAULT}
+read -p "Enter path to keep-core config files directory [$CONFIG_DIR_PATH_DEFAULT]: " config_dir_path
```

Comment:
I think we need to do the same in `install.sh` and keep config files in `configs` directory by default just like we do for `keep-ecdsa`. This approach is much more convenient. A single `./install.sh` modifies all the configuration files, and then we can start any number of clients with `./start.sh` without any additional work. 

The current workflow is weird: we first modify a single config file and then select this config file again when starting.

Reply:
okay, done